boards/arm/stm32wb5mmg/doc/stm32wb5mmg.rst

```diff
@@ -0,0 +1,303 @@
+   and Bluetooth|reg| Low Energy
+ - 48-bit EUI
+
+More information about STM32WB55RG can be found here:
+
+- `STM32WB5MMG on www.st.com`_
+- `STM32WB5MMG datasheet`_
+
+Supported Features
+==================
+
+The Zephyr STM32WB5MMG board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| RADIO     | on-chip    | Bluetooth Low Energy                |
++-----------+------------+-------------------------------------+
+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/stm32wb5mmg/stm32wb5mmg_defconfig``
+
+Bluetooth and compatibility with STM32WB Copro Wireless Binaries
+================================================================
+
+To operate bluetooth on STM32WB5MMG, Cortex-M0 core should be flashed with
+a valid STM32WB Coprocessor binaries (either 'Full stack' or 'HCI Layer').
+These binaries are delivered in STM32WB Cube packages, under
+Projects/STM32WB_Copro_Wireless_Binaries/STM32WB5x/
+For compatibility information with the various versions of these binaries,
+please check `modules/hal/stm32/lib/stm32wb/hci/README <https://github.com/zephyrproject-rtos/hal_stm32/blob/main/lib/stm32wb/hci/README>`__
+in the hal_stm32 repo.
+Note that since STM32WB Cube package V1.13.2, "full stack" binaries are not compatible
+anymore for a use in Zephyr and only "HCI Only" versions should be used on the M0
+side.
+
+Connections and IOs
+===================
+
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+.. rst-class:: rst-columns
+
+- UART_1 TX/RX : PB7/PB6
+- LPUART_1 TX/RX : PA3/PA2
+- USB : PA11/PA12
+- SWD : PA13/PA14
+
+System Clock
+------------
+
+STM32WB5MMG System Clock could be driven by internal or external oscillator,
+as well as main PLL clock. By default System clock is driven by HSE clock at 32MHz.
+
+Serial Port
+-----------
+
+STM32WB5MMG board has 2 (LP)U(S)ARTs. The Zephyr console output is assigned to USART1.
+Default settings are 115200 8N1.
+
+
+Programming and Debugging
+*************************
+
+Applications for the ``stm32wb5mmg`` board configuration can be built the
+usual way (see :ref:`build_an_application`).
+
+Flashing
+========
+
+STM32WB5MMG needs an external ST-Link for flashing the firmware. If you are using
+one of the  ``b_u585i_iot02a`` or ``nucleo_wb55rg`` boards, you can use the on-board
+ST-Link to flash the  STM32WB5MMG module.
+
+If you prefer, you can use pyocd, but it requires to enable "pack" support with
+the following pyocd command:
```

Comment:
Don't mix probes and flashing tools.
Pyocd is not an alternative to ST-Link. It is an alternative to openocd or STM32CubeProgrammer.